Gemfile

```diff
@@ -41,7 +41,9 @@ end
 
 # ActiveSupport
 gem 'dalli', '>= 2.2.1'
-
+group :test do
+  gem 'listen', '~>1.0'
```

Comment:
On the contrary, when you start a project you want to depend on the most recent version (3.0.1). That is two major versions ahead!